pkgs/development/libraries/mapbox-gl-native/default.nix

```diff
@@ -0,0 +1,33 @@
+{ lib, mkDerivation, fetchFromGitHub, cmake, pkg-config
+, qtbase, curl, libuv, glfw3 }:
+
+mkDerivation rec {
+  pname = "mapbox-gl-native";
+  version = "2020.06.07";
```

Comment:
That version number is actually invalid in nixpkgs. It should be unstable-2020-06-07.